pkg/kubelet/images/image_gc_manager.go

```diff
@@ -39,6 +39,10 @@ import (
 	"k8s.io/kubernetes/pkg/kubelet/util/sliceutils"
 )
 
+var (
+	errNoImageToCleanup = goerrors.New("NoImageToCleanup")
```

Comment:
I'm not seeing a reason to have this look like an event type. Maybe it would be better as "no images to cleanup" and then the error condition below would read more like a sentence?